typings/index.d.ts

```diff
@@ -0,0 +1,1909 @@
+
+  export { default as TextField } from 'material-ui/TextField';
+
+  export { default as Toolbar } from 'material-ui/Toolbar';
+}
+
+/* ============================================= */
+/*                                               */
+/*                   COMPONENTS                  */
+/*                                               */
+/* ============================================= */
+
+declare module 'material-ui/AppBar' {
+  export { default } from 'material-ui/AppBar/AppBar';
+}
+
+declare module 'material-ui/AppBar/AppBar' {
+  export interface AppBarProps {
+    color?: MaterialUI.PropTypes.Color;
+    position?: 'static' | 'fixed' | 'absolute';
+  }
+
+  export default class AppBar extends MaterialUI.Component<AppBarProps> {}
+}
+
+declare module 'material-ui/Avatar' {
+  export { default } from 'material-ui/Avatar/Avatar';
+}
+
+declare module 'material-ui/Avatar/Avatar' {
+  export interface AvatarProps {
+    alt?: string;
+    childrenClassName?: string;
+    component?: React.ReactNode;
+    imgProps?: Object;
+    sizes?: string;
+    src?: string;
+    srcSet?: string;
+  }
+
+  export default class Avatar extends MaterialUI.Component<AvatarProps> {}
+}
+
+declare module 'material-ui/Badge' {
+  export { default } from 'material-ui/Badge/Badge';
+}
+
+declare module 'material-ui/Badge/Badge' {
+  export interface BadgeProps {
+    badgeContent: React.ReactNode;
+    children: React.ReactNode;
+    color?: 'default' | 'primary' | 'accent';
+  }
+
+  export default class Badge extends MaterialUI.Component<BadgeProps> {}
+}
+
+declare module 'material-ui/BottomNavigation' {
+  export { default } from 'material-ui/BottomNavigation/BottomNavigation';
+  export {
+    default as BottomNavigationButton,
+  } from 'material-ui/BottomNavigation/BottomNavigationButton';
+}
+
+declare module 'material-ui/BottomNavigation/BottomNavigation' {
+  export interface BottomNavigationProps {
+    children: React.ReactNode;
+    onChange?: React.ReactEventHandler<any>;
+    showLabels?: boolean;
+    value?: any;
+  }
+
+  export default class BottomNavigation extends MaterialUI.Component<
+    BottomNavigationProps
+  > {}
+}
+
+declare module 'material-ui/BottomNavigation/BottomNavigationButton' {
+  export interface BottomNavigationButtonProps {
+    icon?: React.ReactNode;
+    label?: React.ReactNode;
+    onChange?: (event: React.ChangeEvent<{}>, value: number) => void;
+    onClick?: React.ReactEventHandler<any>;
+    selected?: boolean;
+    showLabel?: boolean;
+    value?: number;
+  }
+
+  export default class BottomNavigationButton extends MaterialUI.Component<
+    BottomNavigationButtonProps
+  > {}
+}
+
+declare module 'material-ui/Button' {
+  export { default } from 'material-ui/Button/Button';
+}
+
+declare module 'material-ui/Button/Button' {
+  export interface ButtonProps {
+    color?: MaterialUI.PropTypes.Color | 'contrast';
+    component?: React.ReactNode;
```

Comment:
This isn't a node, it's a component ( string e.g. 'div' or function ) **not** element or node.

Change it to `React.SFC | React.ComponentClass | string` ( I think )